cli/tests/unit/data/node.txt

```diff
@@ -1,2 +1,2 @@
-HOSTNAME       IP                          ID                    
-dcos-01   172.17.8.101  20150630-004309-1695027628-5050-1649-S0  
\ No newline at end of file
+HOSTNAME       IP                          ID                    TYPE   
+dcos-01   172.17.8.101  20150630-004309-1695027628-5050-1649-S0  slave  
```

Comment:
Right. `slave` should be `agent`. But I thought `master` was okay because in my mind we need to distinguish between what I think of as multiple masters and the one leader (or leading master). Or have I got the terminology wrong?